test/e2e/run.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -e
+
+export NOTATION_E2E_BINARY_PATH=$(realpath $1)
+if [ ! -f "$NOTATION_E2E_BINARY_PATH" ];then
+    echo "run.sh <notation-binary-path>"
+    exit 1
+fi
+
+# install dependency
+go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo
+
+# set environment variable for E2E testing
```

Comment:
lgtm. 

Although I hope you are still going to do something like the following (perhaps in a different test), so you are not tied only to zot.

```
registries := {"zot": {setup, teardown}, "docker": {setup, teardown}, ... }

for k,v := range registries {
  v.setup()
  run notation tests on registry "k"
  v.teardown()
}  
```

Reply:
Thanks! Multi-registry support may be implemented in the following PRs.